turbo.json

```diff
@@ -12,73 +12,17 @@
       "dependsOn": ["^build"],
       "outputMode": "new-only"
     },
-    "@audius/compose#start": {
```

Comment:
cool so opting to put them in turbo files in the proper location basically?

Reply:
Yup! I didn't know this was a thing before but it's way cleaner